src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/WatchImage.cs

```diff
@@ -41,7 +41,7 @@ public void CustomizeView(WatchImageCore nodeModel, NodeView nodeView)
 
         private void NodeModelOnPropertyChanged(object sender, PropertyChangedEventArgs args)
         {
-            if (args.PropertyName != "IsUpdated") 
+            if (args.PropertyName != "CachedValue") 
```

Comment:
This component is erroneously using `IsUpdated`, which could potentially cause it to initiate multiple rapid and potentially unnecessary updates.